pkg/name/name_test.go

```diff
@@ -0,0 +1,63 @@
+package name_test
+
+import (
+	"fmt"
+
+	"github.com/rancher/fleet/pkg/name"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Name", func() {
```

Comment:
nit: It is great that the `HelmReleaseName` is now in the `name` package and that we have tests for it!
However I think in this case it would be better to have normal go unit tests and not use Ginkgo here. I think it doesn't add much value here and add unnecessary complexity. I would leave Ginkgo for integration and e2e tests.

This is not a blocker, just my opinion on this. Feel free merge it as it is

Reply:
Yes true, especially since I use a test with a table, but I'd normally use ginkgo for all tests. It's just that the few existing unit tests are old.